From: olh@suse.de
Subject: update for recent gas macro handling changes

as -a32 drivers/macintosh/mol/_traps.o.s  -o drivers/macintosh/mol/_traps.o
drivers/macintosh/mol/asm-files/entry.S: Assembler messages:
drivers/macintosh/mol/asm-files/entry.S:58: Error: too many positional arguments
drivers/macintosh/mol/asm-files/splitmode.S:348: Error: too many positional arguments
make[3]: *** [drivers/macintosh/mol/_traps.o] Error 1

Index: linux-2.6.13/drivers/macintosh/mol/asm-files/entry.S
===================================================================
--- linux-2.6.13.orig/drivers/macintosh/mol/asm-files/entry.S
+++ linux-2.6.13/drivers/macintosh/mol/asm-files/entry.S
@@ -55,7 +55,7 @@ mac_entry_initialize:
 	mtcr	r3
 
 	// msr to be used in mac-mode
-	LOADI	r3,MSR_ME | MSR_IR | MSR_DR | MSR_PR | MSR_EE 
+	LOADI	r3,(MSR_ME | MSR_IR | MSR_DR | MSR_PR | MSR_EE )
 	stw	r3,K_MSR(r1)
 
 	PERF_MONITOR_SETUP /**/ r4
Index: linux-2.6.13/drivers/macintosh/mol/asm-files/splitmode.S
===================================================================
--- linux-2.6.13.orig/drivers/macintosh/mol/asm-files/splitmode.S
+++ linux-2.6.13/drivers/macintosh/mol/asm-files/splitmode.S
@@ -331,7 +331,7 @@ splitm_fallback:
 	sync					// 74xx needs this
 
 	mtsrr0	r2				// The simplest thing is to do an RFI
-	LOADI	r3,MSR_EE | MSR_PR | MSR_IR | MSR_SE | MSR_BE
+	LOADI	r3,(MSR_EE | MSR_PR | MSR_IR | MSR_SE | MSR_BE)
 	andc	r4,r7,r3			// Clear msr bits (r7=srr1)
 	xGPR_LOAD	R6
 	xGPR_LOAD	R7	
